src/main_train.py

```
@@ -0,0 +1,219 @@
+    parser.add_argument("--skip_shap", action="store_true", help="SHAP 분석 스킵 (빠른 검증)")
+    args = parser.parse_args()
+
+    # ── 설정 + 로깅 ─────────────────────────────────────────
+    cfg = load_config(args.config)
+    setup_logging(cfg["logging"]["level"], cfg["logging"].get("log_file"))
+    logger = logging.getLogger("main_train")
+
+    # ══════════════════════════════════════════════════════════════
+    # 1. 데이터 로드 + 검증 + 분할
+    # ══════════════════════════════════════════════════════════════
+    X, y, treatment, cid, feat_names = load_dataset(
```

Ruff (0.15.12)

[warning] Unpacked variable `feat_names` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)